numba/tests/test_array_manipulation.py

```diff
@@ -48,6 +48,10 @@ def numpy_array_reshape(arr, newshape):
     return np.reshape(arr, newshape)
 
 
+def numpy_broadcast_to(arr, shape):
+    return np.broadcast_to(arr, shape)
```

Comment:
Do these tests need a copy of `np.broadcast_to`?

Reply:
What do you mean?

Reply:
It seems like one could replace `pyfunc` and `cfunc` below with something like
```python
pyfunc = np.broadcast_to
cfunc = jit(nopython=True)(lambda arr, shape: pyfunc(arr, shape))
```
then there's no need for a function like `numpy_broadcast_to`&mdash;unless I'm missing something.